components/brave_perf_predictor/browser/perf_predictor_tab_helper.cc

```diff
@@ -109,8 +110,7 @@ void PerfPredictorTabHelper::RecordSavings() {
       if (bandwidth_tracker_)
         bandwidth_tracker_->RecordSavings(savings);
 #if defined(OS_ANDROID)
-        chrome::android::BraveShieldsContentSettings::DispatchSavedBandwidth(
-          savings);
+      prefs->SetUint64(prefs::kLastBandwidthSavedBytes, savings);
```

Comment:
Just curiosity, why android wants last savings?
c++ and android uses different bandwidth tracker?

Reply:
@simonhong we have kBandwidthSavedBytes which accumulates saved bandwidth data and for android we need to have saved bandwidth data for each session for privacy report feature.

Reply:
@simonhong I asked the same thing, there is a whole different UI on android that has bandwidth by week, month, etc...

Reply:
@simonhong @bridiver i have added comment for the pref use.